tools/windows-node-installer/test/e2e/utils.go

```diff
@@ -0,0 +1,42 @@
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/masterzen/winrm"
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/types"
+	"strconv"
+	"time"
+)
+
+const (
+	// winRMPort is the secure WinRM port
+	winRMPort = "5986"
+)
+
+var (
+	// stores windows instance credential information.
+	credentials *types.Credentials
+	user        = "Administrator"
+)
+
+// setupWinRMClient initializes the winRM client and executes commands on the windows node
+func setupWinRMClient(command string) error {
+	stdout := new(bytes.Buffer)
+	stderr := new(bytes.Buffer)
+	winRMPortNo, err := strconv.Atoi(winRMPort)
+	if err != nil {
+		return fmt.Errorf("failed to convert to an integer with error: %v", err)
+	}
+	endpoint := winrm.NewEndpoint(credentials.GetInstanceId(), winRMPortNo, true, true,
+		nil, nil, nil, time.Minute*1)
+	winrmClient, err := winrm.NewClient(endpoint, user, credentials.GetPassword())
+	if err != nil {
+		return fmt.Errorf("failed to initialize winRM client with error: %v", err)
+	}
+	_, err = winrmClient.Run(command, stdout, stderr)
+	if err != nil {
```

Comment:
To be doubly sure, please check that `stderr` does not return anything. This should be done at the call site.